From d1c6b7242e843843fbf6c961047440c7983b142f Mon Sep 17 00:00:00 2001
Message-Id: <d1c6b7242e843843fbf6c961047440c7983b142f@dist-git>
From: Laine Stump <laine@laine.org>
Date: Thu, 11 Apr 2019 15:14:31 -0400
Subject: [PATCH] qemu_hotplug: rename a virDomainDeviceInfoPtr to avoid
 confusion

Having an InfoPtr named "dev" made my brain hurt. Renaming it to
"info" gives one less thing to confuse when looking at the code.

Signed-off-by: Laine Stump <laine@laine.org>
ACKed-by: Peter Krempa <pkrempa@redhat.com>
(cherry picked from commit 1c2866a1f6087837688c3c2beea08753dc6871d0)

Partially-Resolves: https://bugzilla.redhat.com/1658198
Signed-off-by: Laine Stump <laine@redhat.com>
Signed-off-by: Laine Stump <laine@laine.org>
Message-Id: <20190411191453.24055-20-laine@redhat.com>
Acked-by: Michal Privoznik <mprivozn@redhat.com>
---
 src/qemu/qemu_hotplug.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/qemu/qemu_hotplug.c b/src/qemu/qemu_hotplug.c
index 39e40e2740..2bfb5f8d54 100644
--- a/src/qemu/qemu_hotplug.c
+++ b/src/qemu/qemu_hotplug.c
@@ -3784,12 +3784,12 @@ static int qemuComparePCIDevice(virDomainDefPtr def ATTRIBUTE_UNUSED,
 }
 
 static bool qemuIsMultiFunctionDevice(virDomainDefPtr def,
-                                      virDomainDeviceInfoPtr dev)
+                                      virDomainDeviceInfoPtr info)
 {
-    if (dev->type != VIR_DOMAIN_DEVICE_ADDRESS_TYPE_PCI)
+    if (info->type != VIR_DOMAIN_DEVICE_ADDRESS_TYPE_PCI)
         return false;
 
-    if (virDomainDeviceInfoIterate(def, qemuComparePCIDevice, dev) < 0)
+    if (virDomainDeviceInfoIterate(def, qemuComparePCIDevice, info) < 0)
         return true;
     return false;
 }
-- 
2.21.0

